code/game/objects/items/storage/backpack.dm

```diff
@@ -231,6 +231,56 @@
 	desc = "A spacious backpack with lots of pockets, worn by Engineers of an Emergency Response Team."
 	icon_state = "ert_engineering"
 
+/obj/item/storage/backpack/welding
+	name = "welding fuel backpack"
+	desc = "A specialized backpack designed to store welding fuel. Be wary of lit welders!"
+	icon_state = "weldpack"
+	var/tank_volume = 250
+	var/reagent_id = /datum/reagent/fuel
+
+/obj/item/storage/backpack/welding/Initialize()
+	create_reagents(tank_volume, DRAINABLE | AMOUNT_VISIBLE)
+	if(reagent_id)
+		reagents.add_reagent(reagent_id, tank_volume)
+	. = ..()
+
+/obj/item/storage/backpack/welding/proc/boom()
+	visible_message("<span class='danger'>\The [src] ruptures!</span>")
+	chem_splash(loc, 5, list(reagents))
+	qdel(src)
```

Comment:
Isn't this version of this proc kinda pointless since the bottom one instantly overrides it? Would make more sense to remove this one and make the other one the original. Or am i missing something here?

Reply:
I believe this is for waterpacks, which is the parent of the welderpack. I can *probably* get away with removing it

Reply:
Yeah this is from `/obj/structure/reagent_dispensers` and seems to be useless since, like i said, it gets overwritten immediately, so this should be safe to remove